### PR TITLE
Deprecate lxd_volume_container_attach

### PR DESCRIPTION
**Pull request by jtopjian**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -354,13 +354,16 @@ resource "lxd_container" "container1" {
...
-  volume_name = "${lxd_volume.volume1.name}"
-  container_name = "${lxd_container.container1.name}"
-  path = "/tmp"
+  device {
+    name = "volume1"
+    type = "disk"
+    properties {
+      path = "/mount/point/in/container"
+      source = "${lxd_volume.volume1.name}"
+      pool = "${lxd_storag_pool.pool1.name}"
```
Comment (shantanugadgil) [inline README.md]: lxd_storag_pool.pool1.name should be lxd_storag**e**_pool.pool1.name

**Comment by jtopjian** [comment]
@shantanugadgil Nice catch - thank you :)

**Comment by jtopjian** [comment]
With #114 merged, I will update this PR to reflect the new docs. Stay tuned.

**Comment by jtopjian** [comment]
Rebased

**Comment by jtopjian** [comment]
Just a heads up that the Travis failures are intermittent. Not an actual problem with the code.

**Pull request merged**